pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/Tissue.java

```diff
@@ -97,9 +99,11 @@ public class Tissue {
     @DbDateConversion (SqlDateConverter.STRING_DAY)
     private String scrollsReceived;
 
+    @UniqueField(true)
     @ColumnName (DBConstants.SK_ID)
     private String skId;
 
+    @UniqueField(true)
     @ColumnName (DBConstants.SM_ID)
     private String smId;
```

Comment:
These 3 fields are the only fields that need unique value checks. I added a new annotation so we know them and we can ignore the values from deleted tissues when we are checking the uniqueness of the input. For example, if a tissue gets deleted and wants to get reentered, crc should be able to enter the sm id again